CoffeeChat/Cells/DayCollectionViewCell.swift

```diff
@@ -0,0 +1,85 @@
+//
+//  DayCollectionViewCell.swift
+//  CoffeeChat
+//
+//  Created by HAIYING WENG on 2/24/20.
+//  Copyright © 2020 cuappdev. All rights reserved.
+//
+
+import UIKit
+
+class DayCollectionViewCell: UICollectionViewCell {
+
+    private let dayBackgroundView = UIView()
+    private let dayLabel = UILabel()
+    private let indicatorView = UIView()
+    
+    private let backgroundViewSize = CGSize(width: 36, height: 36)
+    private let indicatorViewSize = CGSize(width: 8, height: 8)
```

Comment:
i would just use these directly inline since you only use it once.

Reply:
I actually used it in both `init`(for cornerRadius) and `setupConstraints`